src/main/app/drafts/models/claimant.ts

```diff
@@ -0,0 +1,12 @@
+import { ClaimantType } from '../../forms/models/claimantType'
```

Comment:
Please don't use a relative path here, access it via one of the mappings from ts config.